python/F3DPythonBindings.cxx

```diff
@@ -268,9 +268,12 @@ PYBIND11_MODULE(pyf3d, module)
     .def("pan", &f3d::camera::pan, py::arg("right"), py::arg("up"), py::arg("forward") = 0.0)
     .def("zoom", &f3d::camera::zoom)
     .def("roll", &f3d::camera::roll)
-    .def("azimuth", &f3d::camera::azimuth)
-    .def("yaw", &f3d::camera::yaw)
-    .def("elevation", &f3d::camera::elevation)
+    .def("azimuth", &f3d::camera::addAzimuth)
```

Comment:
Let's change the python api name to add_azimuth too

Reply:
Changed.